src/actions/actions.ts

```diff
@@ -2190,11 +2188,11 @@ class MoveToMatchingBracket extends BaseMovement {
       let matchedPosition: Position | undefined = undefined;
 
       for (const { char, pos } of Position.IterateDocument(position, toFind.nextMatchIsForward)) {
-        if (char === charToMatch) {
+        if (char === charToMatch && charToMatch !== toFind.match) {
           stackHeight++;
         }
 
-        if (char === this.pairings[charToMatch].match) {
+        if (char === toFind.match) {
```

Comment:
All of the callers of this function (`this.nextBracket`) are already passing an object from `this.pairings[char]` as the `toFind`, so I think this is more correct. This also allows callers of this function to use characters that don't exist in `this.pairings`, such as `"` and `'`